app/main/forms/direct_award_forms.py

```diff
@@ -14,3 +15,14 @@ class CreateProjectForm(FlaskForm):
             Optional()
         ]
     )
+
+class DidYouAwardAContractForm(FlaskForm):
+    did_you_award_a_contract = RadioField(
+        "Did you award a contract?",
+        validators=[InputRequired(message="You need to answer this question.")],
```

Comment:
I talked to @karlchillmaid about this this morning, he said that since the phrase is copied from another page he was okay with us keeping it as is for now for consistency.

However, he also said that it was a good question, and was going to bring it up at content crit to see if people had any thoughts on error messages, and possibly we will revisit this question across Digital Marketplace at a later time.

tl;dr let's merge it!